src/firegraph/CacheManager.ts

```diff
@@ -0,0 +1,46 @@
+interface CacheManagerListener{
```

Comment:
Looks like there are some formatting issues. Would you be willing to add `prettier` as a dev dependency and run it?

Reply:
Sure!